google-chart.html

```diff
@@ -263,6 +279,12 @@
                           { selection: this.chartObject.getSelection() });
                   }.bind(this));
 
+              if (this.materialDesign){
+                if (this.type=='bar'){
+                  this.options.bars='horizontal';
+                }
+                this.options=google.charts.Bar.convertOptions(this.options);
```

Comment:
My bad, followed the sample in the [documentation](https://developers.google.com/chart/interactive/docs/gallery/columnchart#Material) which is using the `google.charts.Bar` object but a bit later is suggested to use `google.charts.Column`.